package.json

```diff
@@ -12,5 +11,10 @@
   },
   "devDependencies": {
     "lerna": "^3.22.1"
+  },
+  "dependencies": {
+    "pg": "^8.3.0",
+    "pg-hstore": "^2.3.3",
+    "sequelize": "^6.3.4"
```

Comment:
There are actually three separate package.json files in this repo. The `api` project has one, as does the `app` project. These are two independent applications that we're managing in one repo (a pattern called a monorepo). There's also this one in the root, but this one is just for helping us manage the two separate packages.

Given that, what do you think about removing these dependencies and deleting the root node_modules folder? Then these dependencies could be installed from the api folder instead.

Reply:
Package has been moved